servers/src/main/java/tachyon/master/block/BlockMaster.java

```diff
@@ -607,6 +607,45 @@ private BlockInfo generateBlockInfo(MasterBlockInfo masterBlockInfo) {
   }
 
   /**
+   * This method should only be used by unit tests.
+   *
+   * @param worker a {@link MasterWorkerInfo} to add to the list of lost workers
+   */
+  void addLostWorker(MasterWorkerInfo worker) {
```

Comment:
Could you take a look at the class Tester and BlockDataManager.PrivateAccess to see how we use the private access pattern for adding methods only used by unit tests?